internal/cmd/cmd.go

```diff
@@ -21,6 +23,25 @@ var (
 	errParsing       = errors.New("parsing error")
 )
 
+func init() {
+	// NOTE: this is mostly to set up logging in the case where
+	// the command doesn't exist or the construction of the command
+	// errors out before the PersistentPreRunE setup in the below function.
+	// It helps keep log output visually consistent for a user even in
+	// exceptional cases.
+	var output io.Writer
+
+	if isatty.IsTerminal(os.Stdout.Fd()) {
```

Comment:
should this check `Stderr` since that's where we're writing to?

Reply:
I cribbed from the [cobrazerolog](https://github.com/jzelinskie/cobrautil/blob/main/cobrazerolog/cobrazerolog.go#L93-L98) source, so I'll update it there if necessary.

I think that would technically be safer, but I also can't think of a situation where you'd have a TTY file descriptor on stdout but not stderr.

Reply:
@jzelinskie thoughts here since you wrote this in cobrazerolog?

I wouldn't block on this but if it makes sense to change we should change both places.